tensorflow/core/kernels/sparse/mat_mul_op.cc

```diff
@@ -38,7 +38,6 @@ limitations under the License.
 #include "tensorflow/core/platform/threadpool.h"
 
 #if GOOGLE_CUDA || TENSORFLOW_USE_ROCM
-#include "third_party/gpus/cuda/include/cuda.h"
```

Comment:
thanks for fixing this! I think this is due to https://github.com/tensorflow/tensorflow/pull/63030/files#diff-697f93c85de88980025959c0e5ab71769fae19ae8dcb10bf923a7f610741009bR41  could you create a PR to upstream fix it? (although it might take a while to be merged in TF repo...) I think it needs go to `#if GOOGLE_CUDA` only.

Reply:
Created this PR for upstream
https://github.com/tensorflow/tensorflow/pull/64038